impl/src/test/java/org/ehcache/impl/internal/statistics/DefaultTierStatisticsTest.java

```diff
@@ -47,7 +48,7 @@
   public void before() {
     CacheConfiguration<Long, String> cacheConfiguration =
       CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class,
-        newResourcePoolsBuilder().heap(10))
+        newResourcePoolsBuilder().heap(10, EntryUnit.ENTRIES))
```

Comment:
Why?

Reply:
Another IntelliJ warning about calling a static method from an instance. I agree with it but preferred to use this version of `heap` because when discussing it was agreed that the static version is probably a mistake in the API.